Packs/LogPoint_SIEM_Integration/Integrations/LogPoint_SIEM_Integration/README.md

```diff
@@ -1047,9 +1049,10 @@ Gets the search ID based on the provided search parameters.
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
 | query | LogPoint search query. | Required | 
-| time_range | Time range. For example: Last 30 minutes, Last 7 days, etc. If not provided, it will use 'Last 5 minutes' as the time range by default. Default is Last 5 minutes. | Optional | 
-| limit | Number of logs to fetch. If not provided, first 100 logs will be displayed. Default is 100. | Optional | 
-| repos | Comma-separated list of LogPoint repos from which logs are to be fetched. If not provided, it will display logs from all repos. | Optional | 
+| time_range | Time range. For example: Last 30 minutes, Last 7 days, etc. If not provided, it will use 'Last 5 minutes' as the time range by default. Default is "Last 5 minutes". | Optional | 
+| limit | Number of logs to fetch. If not provided, the first 100 logs will be displayed. Default is 100. | Optional | 
+| repos | A comma-separated list of LogPoint repos from which logs are to be fetched. If not provided, it will display logs from all repos. | Optional |
+| timeout | LogPoint search timeout. Default is 60. | Optional | 
```

Comment:
@MohamedDoumbouya Is the timeout in seconds?

Reply:
@anas-yousef , yes it's in seconds.